### DISCUSSION TITLE
AGU honors CF with Open Science Recognition Prize

### DISCUSSION THREAD

**Discussion by ethanrd** [cf-convention/discuss#381] [Announcements relevant to CF]
Hi all,

As you may have seen, AGU is honoring CF with a 2024 Open Science Recognition Prize.

Congratulations! To all who have contributed to maintaining and improving CF over the years!

There will be an Honors Ceremony (open to the public) and an Honors Banquet (ticketed) on 11 Dec 2024 (Wednesday) at the Fall AGU meeting, 9-13 December 2024 in Washington DC. And I'm hoping to organize a less formal gathering as well.

If you will be at AGU24, please let me know (you can find my email on my GitHub profile). If you plan to attend the Honors Banquet, let me know that as well. We can reserve a table or two (up to 16 people) at the banquet, I will need the names of attendees for the reserved table(s).

And again, congrats!

Ethan

**Comment by taylor13** [comment]
Yes, congratulations to all, and let me also extend my thanks for your contributions to CF.

A word about those listed on the prize...  AGU rules limit the size of the group to no more than 50.  Originally 62 individuals were nominated, but the list was reduced to 49, which resulted in listing: 

- all authors
- current members of the Conventions & Standard Names Committee
- current members of the Information Management and Support Team
- current and former members of the Governance Panel

The intention was to honor the entire CF community, including the 13 individuals who contributed significantly in the past, but have found better ways to spend their time.  Those left off the list should know that they are considered full co-recipients of the prize.

best regards,
Karl
